src/v2/cookbook/packaing-sfc-for-npm.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Packaging Vue Components for npm
+type: cookbook
+order: 10
+---
+
+## Base Example
+
+Vue components by nature are meant to be re-used. This is easy when the component is only used within a single application. But how can you write a component once and use it in multiple sites/applications? Perhaps the easiest solution is via npm.
+
+By packaging your component to be shared via npm, it can be imported/required into a build process for use full-fledged web applications:
+
+```js
+import MyComponent from 'my-component';
+
+export default {
+  components: {
+    MyComponent,
+  },
+  // rest of the component
+}
+```
+
+Or even used via `<script>` tag in the browser directly:
+
+```html
+  <script src="https://unpkg.com/vue"></script>
+  <script src="https://unpkg.com/my-component"></script>
+  ...
+  <my-component></my-component>
+  ...
+```
+
+Not only does this help you avoid copy/pasting components around, but it also allows you to give back to the Vue community!
+
+## Can't I Just Share .vue Files Directly?
+
+Vue already allows components to be written as a single file. Because a Single File Component (SFC) is already just one file, you might ask:
+
+> "Why can't people use my .vue file directly? Isn't that the simplest way to share components?"
+
+It's true, you could share .vue files directly, and as long as the [Vue build](https://vuejs.org/v2/guide/installation.html#Explanation-of-Different-Builds) being used contains the Vue compiler, that would be enough. For example, string concatenation is used as an optimization in the SSR build, so the .vue file might be preferred in this scenario (see [Packaging Components for npm > SSR Usage](#SSR-Usage) for details). But this excludes anyone who wishes to use the component directly in a browser via `<script>` tag, anyone who uses a runtime-only build, or a build process which doesn't understand what to do with .vue files.
```

Comment:
I like how conversational this paragraph is, however the flow is a bit strange. "and as long as the build you are using contains the vue complier, that would be enough."

Later, maybe replacing "But" with "However," at the beginning of the sentence.

Reply:
I will rework this paragraph a little to change the flow.